src/schema/v2/Feature/FeatureType.ts

```diff
@@ -34,5 +37,16 @@ export const FeatureType = new GraphQLObjectType<
         return feature
       },
     },
+    sets: {
```

Comment:
I think this should be a relay connection (so we can properly paginate, etc.).

Our convention has been to make that explicit (so this would be `setsConnection`). We have a few helpers around for converting a gravity list response to a relay connection (see [here](https://github.com/artsy/metaphysics/blob/9f2367240d5ad35c19c3222417d5b4f62939564f/src/schema/v2/partner.ts#L88-L97) for example).

Reply:
👍